tools/test/heuristics/test_heuristics.py

```diff
@@ -6,10 +6,10 @@
 from typing import Any, Dict, Set
 from unittest import mock
 
-REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent
+REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent.parent
```

Comment:
Why did these changes require going up another level?

Reply:
the old repo_root pointed at pytorch/tools isntead of pytorch, I don't know how it worked before this